.github/workflows/velox_be.yml

```diff
@@ -670,3 +673,12 @@ jobs:
         if: ${{ always() }}
         run: |
           $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/clean.sh
+
+  clean-up-tmp:
```

Comment:
@zwangsheng  It seems a bit overkill to add a common job to do cleanup. The github action become complex.. I prefer to revert this change.

<img width="843" alt="image" src="https://github.com/apache/incubator-gluten/assets/12025282/68880945-5a5f-4007-8ed0-81df78a0d2cb">

Reply:
Well, if we want to merge upload action steps, here always will be a new job.

Reply:
IMO, it is acceptable to add a new job to do the cleanup, especially since we will later incorporate the upload operations into this job.

Reply:
I think we should merge upload and cleanup into one job and it seems unnecessary to pull out upload to a new job. There exists many similar jobs in gluten CI, if we want to make it clear, we should consider other jobs.